### PR TITLE
chore: New route ladder header test + stories

### PR DESCRIPTION
**Pull request by hannahpurcell**

### CODE REVIEW

**Comment by joshlarson** [comment]
I know like 9 times out of 10, I'm all for splitting PR's up into smaller units, but I think this is that tenth time. I would find this (and its parent) easier to review if they were merged into one PR.

File: assets/stories/skate-components/newRouteLadderHeader.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import type { Meta, StoryObj } from "@storybook/react"
+
+import { NewHeader as RouteLadderHeader } from "../../src/components/routeLadder"
+
+const meta = {
+  component: RouteLadderHeader,
+  args: {
+    routeName: "39",
+    onClose: () => {},
```
Comment (firestack) [inline assets/stories/skate-components/newRouteLadderHeader.stories.tsx:9]: Could you replace this with `fn()` from `@storybook/test`?
https://storybook.js.org/docs/essentials/actions#via-storybooktest-fn-spy-function

**Comment by hannahpurcell** [comment]
> I know like 9 times out of 10, I'm all for splitting PR's up into smaller units, but I think this is that tenth time. I would find this (and its parent) easier to review if they were merged into one PR.

Agh, I felt like all my PRs lately were too big, but that's fine. I can combine

File: assets/stories/skate-components/newRouteLadderHeader.stories.tsx (hunk 1 above)
Comment (hannahpurcell) [reply assets/stories/skate-components/newRouteLadderHeader.stories.tsx:9]: Agh ok! I hadn't started looking into the change, so you saved me before I had begun the wild goose chase

**Pull request closed without merging**